api/docs/release.dox

```diff
@@ -145,9 +145,6 @@ compatibility changes:
    expression and code relying on this needs to be rewritten.
```

Comment:
> Amends previous commit with doxygen links and a fix for a comment in the public API.

This seems to omit the most important thing here: it fixes a documentation bug for MCXT_NUM_OPMASK_SLOTS.

Reply:
The was the part with "and a fix for a comment in the public API."
I'll re-phrase

Reply:
I thought the comment fix was the non-doxygen becoming doxygen for the new type.